### DISCUSSION TITLE
Le bouton suppr clavier

### DISCUSSION THREAD

**Discussion by FlorentAdell** [witsa/synapps#624] [Ideas]
Bonjour,

Dans la prochaine version de synapps, il serait cool de pouvoir assigner la fonction supprimer au bouton suppr du clavier, et idéalement les boutons F1 à F12 du clavier pour des fonctions supplémentaires

**Comment by Raphwit** [comment]
Hello, effectivement ce serait un plus mais cela cause quelques soucis de design puisque dans Synapp on sélectionne de nombreux éléments simultanément : la scène, l'acteur, etc ...

Il faudrait que l'utilisateur soit parfaitement au courant de ce qu'il va supprimer avec le bouton suppr, car cela peut être irrémédiable, pour les scènes notamment, nous réfléchissons donc encore à l'implémentation de la fonctionnalité proposée et si nous trouvons une solution convenable elle sera effectivement ajoutée !
